app/views/shared/_footer_secondary.html.erb

```diff
@@ -1,9 +1,10 @@
 <div class="l-footer-secondary t-footer-secondary">
-  <div class="footer-secondary--inner">
+  <div class="l-constrained">
+  <div class="l-footer-secondary--inner">
```

Comment:
Minor thing: could you indent this?

Reply:
I will be indenting the entire document just before I merge this PR.